app.js

```diff
@@ -0,0 +1,7 @@
+// Complete the challenge by writing JavaScript below
+var listItems = document.getElementsByTagName('li');
```

Comment:
Loks good!
<img width="391" alt="image" src="https://github.com/user-attachments/assets/acaf1ecd-925e-4e3e-ade3-605a73467fe2">

However
Please avoid using `var` as it's value can be reassigned outside the block scope and this may get problematic when working on a large code base.

**Rule of thumb is**
- Always prioritize using `const` if you know the value won't get reassigned to avoid bugs as it also helps other devs you might be working with in a codebase for readability
- Always use `let` as much as possible than using `var` if you know the value will get reassigned and `let` only works on it's block scope

```suggestion
const listItems = document.getElementsByTagName('li');
```